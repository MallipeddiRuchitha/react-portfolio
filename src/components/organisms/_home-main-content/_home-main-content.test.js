import React from 'react';
import { shallow ,mount} from 'enzyme';
import HomeMainContent from './_home-main-content';

describe(" HomeMainContent component",() =>{
    test("Matches the snapshot",() => {
        const home= shallow(< HomeMainContent/>);
        expect(HTMLOListElement).toMatchSnapshot();

    })
    test("GalleryAdmmin is rendered",() => {
        const home= shallow(< HomeMainContent/>);
        
        expect(home.find('#home').exists()).toEqual(true);
    });
   
   
   

});