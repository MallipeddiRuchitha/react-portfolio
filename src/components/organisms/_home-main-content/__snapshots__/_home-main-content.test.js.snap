// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` HomeMainContent component Matches the snapshot 1`] = `[Function]`;
